jsf-showcase-webapp/pom.xml

```diff
@@ -238,7 +238,7 @@
 		<dependency>
 			<groupId>com.liferay.faces.test</groupId>
 			<artifactId>com.liferay.faces.test.selenium</artifactId>
-			<version>0.4.0</version>
+			<version>0.4.1-SNAPSHOT</version>
```

Comment:
@stiemannkj1 It's 0.4.1-SNAPSHOT in liferay-faces-test-selenium, and I wasn't sure if there would be any changes along the way, especially with the dataTesters being more of a long-term project.